Inventory/Items/BDSM/HypnovisorMk1.gd

```diff
@@ -51,7 +51,7 @@ func isRestraint():
 
 func generateRestraintData():
 	restraintData = RestraintHypnovisor.new()
-	restraintData.setLevel(5)
+	restraintData.setLevel(calculateBestRestraintLevel())
```

Comment:
This item is sold by an NPC in the game, maybe the restrain level should stay a constant?

Reply:
Hypnovisor = dropped item by common NPC's
Hypnovisor**Mk1** = a special item only unlocked and sold after a specific quest.
The **Mk1** variant is **never** used by common NPC's. (Until it's changed at least)

But for selling or specific quest events, I agree a way to force specific level loot could be nice.